### PR TITLE
Fix time.sleep and empty maches

### PR DESCRIPTION
**Pull request by Kulamabada**
Chłopacy naprawiliśmy i działa obiecujemy i zrewjujujcie UWU

### CODE REVIEW

File: scrapper.py [hunk 1]
```diff
@@ -109,18 +113,27 @@ def get_n_recent_matches(self, n: int, player: Player) -> list[Opgg_match]:
...
             pass
-
         # Get matches divs
         matches_div = self.driver.find_elements(
             By.CLASS_NAME,
-            "e1gknzrf0",
+            "e4p6qc61",
+        )
+
+        page = self.driver.find_element(
```
Comment (zielinsky) [inline scrapper.py]: unnecessary lines 129-137

File: scrapper.py [hunk 2]
```diff
@@ -310,11 +325,18 @@ def get_champion_stats(self, champion: Champion, tier: Tier) -> list[Champ_stats
...
         for lane_elem in lane_elements:
             lane_name = lane_elem.get_attribute("data-value")
-            lane_a = lane_elem.find_element(By.TAG_NAME, "a")
-            lane_a.click()
-            time.sleep(
-                3
-            )  # ================================== WTF ========================================
+            if lane_elem != lane_elements[0]:
+                old_lane_flag = self.driver.find_element(By.CLASS_NAME, "ee92tbj0").text.split('.')[0]
+                new_lane_flag = self.driver.find_element(By.CLASS_NAME, "ee92tbj0").text.split('.')[0]
```
Comment (zielinsky) [inline scrapper.py]: new_lane_flag = old_lane_flag or better:
old_lane_flag = new_lane_flag = self.driver.find_element(By.CLASS_NAME, "ee92tbj0").text.split('.')[0]

File: scrapper.py (hunk 1 above)
Comment (Kulamabada) [reply scrapper.py]: deleted

File: scrapper.py (hunk 2 above)
Comment (Kulamabada) [reply scrapper.py]: done better

**Review by zielinsky — APPROVED** [review]

**Pull request merged**